hypervisor/include/arch/x86/irq.h

```diff
@@ -207,6 +209,7 @@ void vcpu_make_request(struct acrn_vcpu *vcpu, uint16_t eventid);
  * @pre vcpu != NULL
  */
 int32_t exception_vmexit_handler(struct acrn_vcpu *vcpu);
+int32_t nmi_window_vmexit_handler(struct acrn_vcpu *vcpu);
```

Comment:
please add "__unused" prefix to keep the consistency.

Reply:
@HuiHuangShi. Actually, we use the parameter "vcpu" in nmi_window_vmexit_handler. But I add the __unused prefix in its implementation by mistake. I have fix it in next version.